sdk/include/psdk/uxtheme.h

```diff
@@ -57,6 +56,8 @@ extern "C" {
 
 typedef HANDLE HPAINTBUFFER;
 typedef HANDLE HTHEME;
+//Vista+
```

Comment:
```suggestion
// Vista+
```
Maybe add a _win32_winnt and/or a dll_export_ ... compile check?

Reply:
This has been talked about off and on
For sure I'm not going to add active checks for vista RTM-SP2 header changes anymore when they don't effect anything.

I think // Vista+ is fair. but honestly If your goal is to retain the ability to compile an NT5.2 OS these changes to the sdk won't effect that :)